portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomFact() {
+  const facts =
+      ['Sometimes I get my age wrong.', 'I\'m learning to Chinese calligraphy.', 'I could speak 5 languages/dialects (technically).', 'A huge sports fan'];
```

Comment:
Please line break code at 80 characters. This code (hopefully) started in [Google JavaScript style](https://google.github.io/styleguide/jsguide.html) so I recommend following that. Also note (per the doc) that we have a tool that can format your code.